native-script.ps1

```diff
@@ -8,6 +8,9 @@
 param(
 	[switch] $SilentMode
 )
+
+$scriptUrl = "https://raw.githubusercontent.com/NativeScript/setup-scripts/master/native-script.ps1"
```

Comment:
This file doesn't exist anymore, actually. The correct one is www.nativescript.org/setup/win